core_data_modules/cleaners/codes/kenya_codes.py

```diff
@@ -342,52 +342,1829 @@ class KenyaCodes(object):
     WUNDANYI = "wundanyi"
     YATTA = "yatta"
 
-    # Kenya Wards
-    # Machakos County Wards, src: https://kenyayote.com/iebc-machakos-county-registered-voters-constituency-wards/
+    # Kenya Wards src: https://docs.google.com/spreadsheets/d/1NQEJ7ikjrmqQAI8sJ-SXyoZhvOYftMtY/edit?usp=share_link&ouid=105008405800104279592&rtpof=true&sd=true
```

Comment:
Can you add another note describing where this spreadsheet was retrieved from or how it was generated?

Reply:
Updated thanks.